src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -19,14 +19,24 @@
 
 class TPkgViewRendererLessCompiler
 {
+    /**
+     * @var string
+     */
+    private $dir;
+
+    public function __construct(string $dir)
```

Comment:
Could you rename this var to resemble the configuration option ($cssOutputDir if you follow my suggestion above)?

Reply:
ok